lib/modules/manager/gitlabci-include/extract.ts

```diff
@@ -33,20 +99,13 @@ export function extractPackageFile(content: string): PackageFile | null {
     const doc: any = load(replaceReferenceTags(content), {
       json: true,
     });
```

Comment:
```suggestion
    const doc: any = load(replaceReferenceTags(content), {
      json: true,
    }) as GitlabPipeline;
```
Have you considered extending the Interface in the [`gitlab-ci`](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gitlabci/types.ts#L5) manager?  

Using interfaces would also simplify your code.

Reply:
no, please use strong types

Reply:
@viceice  Refactored to be strongly typed, ready for re-review